TFT/src/User/Configuration.h

```diff
@@ -244,9 +244,9 @@
  *   on the estimate the slicer has done (see notes below).
  *
  * NOTES:
- *   - Time mode needs info from the slicer such as the elapsed time or the remaining time ("M73 Rxx").
- *     If no "M73 Rxx" is present in the G-code file then it needs the slicer includes the time related
- *     info as comments and "FILE_COMMENT_PARSING" to be enabled.
+ *   - Time mode needs info from the G-code file such as the elapsed time or the remaining time. This info
+ *     can be supplied as "M73 Rxx" G-code or as comment. Both must be generated by the slicer. If comment
+ *     is used than "file_comment_parsing" has to be enabled for it to take effect.
```

Comment:
in Configuration.h you have to use FILE_COMMENT_PARSING not file_comment_parsing (used in config.ini)

Reply:
Fixed. Thank you!